libs/application/templates/children-residence-change/src/fields/components/ContractOverviewAccordion/ContractOverviewAccordion.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Accordion, AccordionItem, Box } from '@island.is/island-ui/core'
+import { CRCApplication } from '../../../types'
+import ContractOverview from '../ContractOverview/ContractOverview'
+
+interface Props {
+  application: CRCApplication
+  title: string
+  id: string
+}
+
+const ContractAccordionOverview = ({ application, title, id }: Props) => {
+  return (
+    <Box paddingX={4} paddingY={2} border="standard">
+      <Accordion singleExpand dividerOnBottom={false} dividerOnTop={false}>
```

Comment:
Since we only have one element there is no need for the `singleExpand` prop here
```suggestion
      <Accordion dividerOnBottom={false} dividerOnTop={false}>
```